### ISSUE TITLE
Can not die after killing by mobs

### ISSUE THREAD

**Issue by ColaLiao** [Cycro54/Revive_Me#3]
![image](https://user-images.githubusercontent.com/16384174/228926994-399438c4-2781-4769-a42e-ad64245aa0a4.png)

![image](https://user-images.githubusercontent.com/16384174/228927493-dc179131-05a5-452f-ad6a-bf4bca9e7c18.png)

**Comment by Cycro54** [comment]
That IS rather bothersome, a quick fix is switching in and out of creative mode. I'll be updating this version soon. Any other mods that could mess with the death system? Also is this a single-player or multiplayer world?

**Comment by ColaLiao** [comment]
after switch creative mode, comes invincible.
single-player world,



[modlist.txt](https://github.com/Cycro54/Revive_Me/files/11118604/modlist.txt)

**Comment by Cycro54** [comment]
I've updated my mod, try downloading the newest version for your mc version, then using /revivemefix if nothing is working
There is also a /reviveme that will fully revive you (in cheat mode only)

**Comment by ColaLiao** [comment]
![image](https://user-images.githubusercontent.com/16384174/229142308-968f5cca-cab4-4ece-ad5d-2129ea9d35fc.png)
still happen

**Comment by ColaLiao** [comment]
1. kill by mob...
2. wait utill time end

**Comment by Cycro54** [comment]
You also tried using /revivemefix too?

**Comment by Cycro54** [comment]
It's going to take forever to download all those mods so if you use Curseforge you can export the profile for the mod and post it here (see screenshots)

I will need your config, mods, saves, and whatever else you may have modified
![1](https://user-images.githubusercontent.com/85908325/229255368-27a434f6-97b8-4ae0-baff-c4fa0d79c66a.png)
![2](https://user-images.githubusercontent.com/85908325/229255375-9da0a31a-d6b7-4a7a-ba28-623c54d55d4f.png)

**Comment by ColaLiao** [comment]
**Also does this happen in every world you create or just the one?**
every world.

https://drive.google.com/file/d/1Xff_yZUiWCmsdH0Hyv8ZKZz1gH6ix2fk/view?usp=sharing

**Comment by Cycro54** [comment]
The modpack you sent me had the old versions of Invocore and reviveme in it. I updated them,and did a couple tests in a flat world. 
I let an iron golem kill me, I typed in the /kill command and I used the /reviveme command as well.
I then opened a regular world, spawned in a husk and allowed that to kill me

It seems like I can't reproduce this error, maybe there is something happening in the debug log that is of use?
You never told me if you used the /revivemefix command either (not /reviveme)


![1](https://user-images.githubusercontent.com/85908325/229274134-f02c03a7-5178-4a4b-974e-449729ab23b2.png)
https://user-images.githubusercontent.com/85908325/229274524-7f89e45f-9295-47da-9558-e378bcf6147f.mp4
https://user-images.githubusercontent.com/85908325/229275488-330922b8-fd74-4659-8f21-8124c4ad7f27.mp4

**Comment by ColaLiao** [comment]
what is /revivemefix do? i didn't try it.

my step:
kill by mob,
wait until time end,
GUI become INF,
switch to creative mode,
use /kill

**Comment by Cycro54** [comment]
It reruns the code that happens when you either revive or go into the dying state. If for some reason you are invulnerable when you come back, typing /revivemefix should at least remove the invulnerability.

I tried those steps you gave me but it still worked every single time

1. Make sure you don't have the old version of the mod still installed
2. Is it possible you sent me an outdated modpack? (it had 192 mods with an old version of reviveme and invocore that I updated)
3. Try posting a video of this occurring. I wanna see the amount of mods you have, the config of revive me, and the steps you take for the mod to glitch
